docs/guides/integration/docker.md

```diff
@@ -59,8 +59,8 @@ If you're using uv to manage your project, you can copy it into the image and in
 ADD . /app
 WORKDIR /app
 
-# Sync the project into a new environment
-RUN uv sync
+# Sync the project into a new environment, using the frozen lockfile
+RUN uv sync --frozen
```

Comment:
should this be `--locked` instead, so as for the, supposedly production, Docker image fail to build if `pyproject.toml` and `uv.lock` are not in sync?

below too?